### PR TITLE
D15184_D16194_Shared_RepositoryChange

### PR DESCRIPTION
**Pull request by prashelke**
Thank you for your contribution.
Before submitting this PR, please make sure:
- [x] PR description and commit message should describe the changes done in this PR
- [x] Verify the PR is point to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [x] Latest Code from master or specific release branch is merged to your branch
- [x] No unwanted\commented\junk code is included
- [x] No new warning upon build solution
- [x] Existing unit test cases are passed
- [x] New Unit tests are added for your development
- [x] Sanity Tests are successfully executed for Existing Functionality
- [x] After creating pull request there should not be any conflicts
- [x] Resolve codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [x] Resolve code review comments

### CODE REVIEW

**Comment by Maheshkale447** [comment]
Please run the complete regression suite to make sure change is not affecting existing business flows

File: Ginger/GingerCoreNETUnitTest/SolutionRepositoryLib/RepositoryTest.cs [hunk 1]
```diff
@@ -1388,5 +1388,24 @@ public void CopyItem_GUIDTest()
            Assert.AreNotEqual(sampleFC.Guid, copiedItemNew.ActFlowControls[0].Guid); 
         }
 
+        [TestMethod]
+
+        public void ActivityGroupTest()
+        {
+            //Arrange
+            //Put the BF in Test Resource having class "GingerCore.DataSource.ActDSConditon" serialized in xml
```
Comment (Maheshkale447) [inline Ginger/GingerCoreNETUnitTest/SolutionRepositoryLib/RepositoryTest.cs:1396]: Please update the comment

File: Ginger/GingerCoreNETUnitTest/TestResources/Repository/Flow 1.Ginger.BusinessFlow.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<GingerRepositoryItem><Header ItemGuid="cf613b63-26ae-47e0-8d3a-0808163e0dce" ItemType="BusinessFlow" CreatedBy="pravishe" Created="202209160601" GingerVersion="4.5.0.0" Version="9" LastUpdateBy="pravishe" LastUpdate="202209210524" />
```
Comment (Maheshkale447) [inline Ginger/GingerCoreNETUnitTest/TestResources/Repository/Flow 1.Ginger.BusinessFlow.xml:2]: Rename the xml to have more meaningful name

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/GingerCoreNETUnitTest/SolutionRepositoryLib/RepositoryTest.cs (hunk 1 above)
Comment (prashelke) [reply Ginger/GingerCoreNETUnitTest/SolutionRepositoryLib/RepositoryTest.cs:1396]: Done

File: Ginger/GingerCoreNETUnitTest/TestResources/Repository/Flow 1.Ginger.BusinessFlow.xml (hunk 2 above)
Comment (prashelke) [reply Ginger/GingerCoreNETUnitTest/TestResources/Repository/Flow 1.Ginger.BusinessFlow.xml:2]: Done

**Pull request merged**